### ISSUE TITLE
C2 Mode296 first

### ISSUE THREAD

**Issue by c72578** [gchudov/cuetools.net#192]
There are a few drives, where C2 `Mode294` is detected but not working properly and `Mode296` has to be used.
This issue contains a build of CUETools, where `Mode296` is tried first in general. This build can be useful for investigating such drive issues.

`Mode294` does not work for these drives and CUETools already contains a specific fix for each of them:
```
ASUS DRW-24D5MT
ASUS DRW-24F1ST d
HL-DT-ST BD-RE BU40N
LG GH24NSD1
PIONEER BDR-XD05
PIONEER BDR-XD07U
PIONEER DVR-S21
Slimtype - DVD A DU8AESH
```

- Fix:
https://github.com/gchudov/cuetools.net/blob/dd0404d1073881b2aae8065961d7300f5ac0f87b/CUETools.Ripper.SCSI/SCSIDrive.cs#L815 (2022-06-02, CUETools 2.2.2)

Further reading concerning C2 error modes:
https://www.cdrinfo.com/Sections/Reviews/Specific.aspx?ArticleId=7378

**Comment by c72578** [comment]
Build of **CUETools 2.2.2** for drive testing, where `Mode296` is tried first in general:
[CUETools_2.2.2_Mode296_first.zip](https://github.com/gchudov/cuetools.net/files/8827459/CUETools_2.2.2_Mode296_first.zip)
`SHA256: fee1efcf3d2388fa9a5fe8d1814710c7a76814614763f0c2e2ae1ffd945fe4d3`

This build has been prepared from the following commit, using GitHub Action release-windows:
https://github.com/c72578/cuetools.net/commit/ac64cab1f416a2aee9b48f4e5c064e88d1817ff6

```diff
diff --git a/CUETools.Ripper.SCSI/SCSIDrive.cs b/CUETools.Ripper.SCSI/SCSIDrive.cs
index 4de60e4..1784ff8 100644
--- a/CUETools.Ripper.SCSI/SCSIDrive.cs
+++ b/CUETools.Ripper.SCSI/SCSIDrive.cs
@@ -811,7 +811,7 @@ namespace CUETools.Ripper.SCSI
                                return true;

                        ReadCDCommand[] readmode = { ReadCDCommand.ReadCdBEh, ReadCDCommand.ReadCdD8h };
-                       Device.C2ErrorMode[] c2mode = { Device.C2ErrorMode.Mode294, Device.C2ErrorMode.Mode296, Device.C2ErrorMode.None };
+                       Device.C2ErrorMode[] c2mode = { Device.C2ErrorMode.Mode296, Device.C2ErrorMode.Mode294, Device.C2ErrorMode.None };
                        // Mode294 does not work for these drives: LG GH24NSD1, ASUS DRW-24D5MT, ASUS DRW-24F1ST d, PIONEER DVR-S21, PIONEER BDR-XD05, PIONEER BDR-XD07U, HL-DT-ST BD-RE BU40N, Slimtype - DVD A DU8AESH. Try Mode296 first
                        // Drives can contain one or multiple spaces in the name, e.g. "ASUS DRW-24F1ST   d". Remove any spaces from Path.
                        string pathNoSpace = Path.Replace(" ", String.Empty);
```

**Comment by c72578** [comment]
A PR, which adds a setting for `C2ErrorMode` is currently in preparation: #203

**Comment by c72578** [comment]
Resolved by #203, #208 and #214.
